.github/mergify.yml

```diff
@@ -0,0 +1,70 @@
+queue_rules:
+  - name: urgent
+    speculative_checks: 2
+    batch_size: 2
+    conditions:
+      - check-success=CI
+      - check-success=Coverage
+
+  - name: medium
+    speculative_checks: 2
+    batch_size: 3
+    conditions:
+      - check-success=CI
+      - check-success=Coverage
+
+  - name: default
+    speculative_checks: 2
+    batch_size: 4
+    conditions:
+      - check-success=CI
+      - check-success=Coverage
+
+pull_request_rules:
```

Comment:
Scope: It seems like we're missing the automated `merge` action.

I think that's a good idea while we are testing mergify, but I just wanted to mention it so other reviewers aren't confused.

We will also need to check the cached state tests in each queue before we auto-merge anything. But that can happen in a later PR.

Reply:
The merge method is automatically considered when a PR is sent into a queue; the default is *merge*

Now, the only extra consideration to take is which Merge method to use. Possible values are **merge**, **squash** or **rebase**.

cc: @dconnolly